inotify/test_support.py

```diff
@@ -11,12 +11,9 @@ def temp_path():
     path = tempfile.mkdtemp()
 
     original_wd = os.getcwd()
-    os.chdir(path)
```

Comment:
Let's leave these in, please.

Reply:
Do you still insist on this? There is an explanation here: https://github.com/dsoprea/PyInotify/pull/48#issuecomment-426215530